.env

```diff
@@ -3,5 +3,6 @@ BROWSER=none
 REACT_APP_ENABLE_GOSUAI=
 REACT_APP_ENABLE_BOTTOM_BANNER=true
 REACT_APP_IMAGE_CDN=https://cdn.cloudflare.steamstatic.com
-REACT_APP_ENABLE_ADSENSE=true
-REACT_APP_ENABLE_DOTA_COACH=true
\ No newline at end of file
+REACT_APP_ENABLE_ADSENSE=
+REACT_APP_ENABLE_DOTA_COACH=
+REACT_APP_STRIPE_PUBLIC_KEY=pk_test_75BQAni4TXzhKT9NiocIaXPz
```

Comment:
Do we need to replace with prod key before shipping?

Reply:
yes